package/MDAnalysis/core/Selection.py

```diff
@@ -543,8 +518,7 @@ class ProteinSelection(Selection):
 
     .. SeeAlso:: :func:`MDAnalysis.lib.util.convert_aa_code`
     """
-    #: Dictionary of recognized residue names (3- or 4-letter).
-    prot_res = {x: None for x in [
+    prot_res = [
```

Comment:
If you want to make it look nicer, create a new class such as `LookupDict` in `lib.util` and use it.